### PR TITLE
Refactored shell scripts. Fix - #91

### PR DESCRIPTION
**Pull request by Pagla-Dasu**
Changes made to `CONTRIBUTING.md` for easy setup for new contributors.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -44,7 +45,7 @@ docker build -t local-kolibri2zim .
 Scrape a channel (here we use the minimal channel, but you could use any other one of interest for your UI developments).
 
 ```
-docker run --rm -it -v $PWD/output:/output local-kolibri2zim kolibri2zim --name "minimal_test" --title "Minimal Kolibri Channel Test" --description "This is a minimal K
+docker run --rm -it -v "$PWD/output:/output" local-kolibri2zim kolibri2zim --name "minimal_test" --title "Minimal Kolibri Channel Test" --description "This is a minimal K
```
Comment (benoit74) [inline CONTRIBUTING.md:48]: I think `docker run --rm -it -v "$PWD/output":/output local-kolibri2zim ...` would be more correct, could you confirm it works as well?

File: CHANGELOG.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix caching of re-encoded video files (#82)
 - Do not start multiple video processing threads by default (`--processes` default value) (second part of #83)
 - Fix logging issue in DEBUG mode
+- Refactored some shell scripts in `CONTRIBUTING.md` (#91)
```
Comment (benoit74) [inline CHANGELOG.md:25]: This has to be added to the `[Unreleased]` section, not to the already released 1.2.0

I would rephrase this to "Enhance contribution guidelines", which is a bit too vague, but more inline with your intent.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -36,6 +36,7 @@ To simplify this, it is possible to:
 - iterate on ZIM UI code
 
 To achieve this, first build the Docker image based on current code base.
+Make sure Docker is running in the background ("Docker Deamon" app for Windows and Mac, or `docker start` in Linux).
```
Comment (benoit74) [inline CONTRIBUTING.md:39]: This should not be placed here, we should instead rewrite the `contributions` section:
- state the requirements more clearly: Python 3.12 and preferably Docker (which engine must run in the background)
- add mention of `pre-commit` which must be activated (code must pass all pre-commit checks to succeed in the CI)

**Review by benoit74 — CHANGES_REQUESTED** [review]
Some fixes, thank you

File: CONTRIBUTING.md (hunk 1 above)
Comment (Pagla-Dasu) [reply CONTRIBUTING.md:48]: Yes, this also works fine. Will refactor this in the next commmit.

File: CONTRIBUTING.md (hunk 3 above)
Comment (Pagla-Dasu) [reply CONTRIBUTING.md:39]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (Pagla-Dasu) [reply CHANGELOG.md:25]: Done

**Comment by Pagla-Dasu** [comment]
@benoit74 , Changes are done according to the request.

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Enhance contribution guidelines: Refactored shell scripts, added additional steps.
```
Comment (benoit74) [inline CHANGELOG.md:9]: Please add the `#91` back, like it was before, at the end of the line

Comment (Pagla-Dasu) [reply CHANGELOG.md]: Done
